### PR TITLE
Issue220

### PR DESCRIPTION
**Pull request by kongliangzhong**

### CODE REVIEW

**Comment by kongliangzhong** [comment]
#220

File: contracts/NameRegistry.sol [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+pragma solidity 0.4.18;
+
+/// @title Ethereum Address Register Contract
+/// @dev This contract maintains a name service for addresses and miner.
```
Comment (dong77) [inline contracts/NameRegistry.sol]: miner and wallet?

File: contracts/NameRegistry.sol [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+/// @author Kongliang Zhong - <kongliang@loopring.org>,
+contract NameRegistry {
+    uint32 id = 0;
```
Comment (dong77) [inline contracts/NameRegistry.sol]: nextId = 1; and use uint64

File: contracts/NameRegistry.sol [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+    struct NameInfo {
+        bytes12 name;
+        uint32  rootId;
+    }
+
+    struct AddressSet {
+        address signer;
+        address feeRecipient;
+        uint32  nextId;
```
Comment (dong77) [inline contracts/NameRegistry.sol]: rename this to `id`

**Pull request merged**